### DISCUSSION TITLE
Updating samagra-comms/uci-console to Angular 16

### DISCUSSION THREAD

**Discussion by abhishekpnt** [samagra-comms/community#78] [General Discussion]
Hi Team,

As we are updating Sunbird-ED portal to Angular 16(https://project-sunbird.atlassian.net/browse/ED-3612), some of the legacy libraries like **ng2-sematic-ui-v9** was causing build issues in portal.
So we have updated **ng2-sematic-ui-v9** to Angular 16 and published with the name **@project-sunbird/ng2-semantic-ui**(https://project-sunbird.atlassian.net/browse/ED-3836)

When we were checking the libraries that were using legacy dependency ng2-sematic-ui-v9, we found that samagra-comms/uci-console is using it and it is causing the issues in the build time of the portal.

So we have removed the **ng2-sematic-ui-v9** and used **@project-sunbird/ng2-semantic-ui** in  samagra-comms/uci-console and updated samagra-comms/uci-console to Angular 16

Please create a branch on https://github.com/samagra-comms/uci-console for pushing these changes and when we raise the PR,
Please merge it and publish the latest package.

CC:  @ChakshuGautam @abhay-samagra  @mohitga @vinukumar-vs @rajeevsathish @princegupta1131

**Comment by apkstreams** [comment]
[Samagra Portal](https://samagra-portal.in/) is working
